services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersValidateSimpleFlowSpec.groovy

```diff
@@ -12,69 +12,64 @@ import groovy.transform.Memoized
 import org.springframework.beans.factory.annotation.Autowired
 import spock.lang.Narrative
 
-@Narrative("""This test suite check the meter validate request on a flow that contains 2+ switches.
-description of fields:
-- missing - those meters/rules, which is NOT exist on a switch, but it is exist in db
+@Narrative("""This test suite check the meter validate feature on a flow that contains 2+ switches.
+Description of fields:
+- missing - those meters/rules, which are NOT present on a switch, but are present in db
 - misconfigured - those meters which have different value (on a switch and in db) for the same parameter
-- excess - those meters/rules, which is exist on a switch, but it is NOT exist in db
-- proper - meters/rules value is the same on a switch and in db
+- excess - those meters/rules, which are present on a switch, but are NOT present in db
+- proper - meters/rules values are the same on a switch and in db
 """)
 class MetersValidateSimpleFlowSpec extends BaseSpecification {
     @Autowired
     SwitchHelper switchHelper
 
-    def "Able to get meter validate information"() {
+    def "Switch validation is able to return correct information about a flow"() {
```

Comment:
This is much better, but too broad as for me. You can name _any_ test in this spec with this name, can't you?

Reply:
Fixed